ipatests/test_integration/test_sssd.py

```diff
@@ -403,3 +403,35 @@ def test_trustdomain_disable_disables_subdomain(self):
                 assert 'no such user' in res.stderr_text
         # verify the user can be retrieved after re-enabling trustdomain
         self.master.run_command(['id', user])
+
+    def test_subdomain_lookup_with_certmaprule_containing_dn(self):
+        """DN names on certmaprules should not break AD Trust lookups.
+
+        Regression test for https://bugzilla.redhat.com/show_bug.cgi?id=1568083
+        """
+        tasks.kinit_admin(self.master)
+
+        # verify the user can be retrieved initially
+        first_res = self.master.run_command(['id', self.users['ad']['name']])
+
+        cert_cn = 'CN=adca'
+        cert_dcs = 'DC=' + ',DC='.join(self.ad.domain.name.split('.'))
+        cert_subject = cert_cn + ',' + cert_dcs
```

Comment:
This can be simplified with :
cert_subj = 'CN=adca,DC=' + ',DC='.join(self.ad.domain.name.split('.'))

Reply:
As we just spoke on IRC, in this case I prefered to split strings in lines and concat after because I though the shorter lines would look nicer.